Poker.BE/Poker.BE.API/Web.config

```diff
@@ -10,67 +10,71 @@
   -->
 <configuration>
```

Comment:
# NO! 👎 
you still had made changes to this file (**a lot**). [you can see it here](https://github.com/WorkshopSE/TexasHoldemPoker.repo/commit/80159863ac4395b0511fcdbd7e77e18c8bc5669f)
please REVERT the file to the my last commit on it:
![image](https://cloud.githubusercontent.com/assets/9889268/26532184/bf65d990-4402-11e7-885a-8a19b9af1540.png)